ReactNativeClient/android/app/build.gradle

```diff
@@ -146,6 +146,7 @@ android {
 }
 
 dependencies {
+    implementation project(':react-native-quick-actions')
```

Comment:
Note to reviewer: I didn't add this manually; it was added automatically when I installed `react-native-quick-actions` via npm and linked the project in xcode (as described in the OP of this PR)